Source/Codable/Codable.swift

```diff
@@ -5,6 +5,7 @@
 //  Created by Edwin Vermeer on 28/06/2017.
 //  Copyright © 2017 EVICT BV. All rights reserved.
 //
+import Foundation
```

Comment:
This worked before because CocoaPods imports bring `Foundation` included. When using SPM, the package needs to be able to be built by itself